clouds/snowflake/modules/sql/quadbin/_FROM_BASE.sql

```diff
@@ -0,0 +1,13 @@
+----------------------------
+-- Copyright (C) 2023 CARTO
+----------------------------
+
+-- change a float or numeri to other base
```

Comment:
`numeric`

Don't know if the name is too generic a and could collide with any other future UDF in the toolbox.